src/components/logs-toolbar.tsx

```diff
@@ -72,12 +73,12 @@ export const LogsToolbar: React.FC<LogsToolbarProps> = ({
     _category: string | ToolbarChipGroup,
     chip: string | ToolbarChip,
   ) => {
-    severityFilter.delete(chip.toString() as Severity);
+    severityFilter?.delete(chip.toString() as Severity);
```

Comment:
Do we need the `?` here, since `severityFilter` now has a default value?

I see that it seems to not be needed in `severityFilter` below?